### PR TITLE
Transfer code from incubator

### PR DESCRIPTION
**Pull request by g-steiner**
**Please check if the PR fulfills these requirements**

- [ ] The commit message follows our guidelines
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

**Does this PR already have an issue describing the problem?**

**What kind of change does this PR introduce?**

**What is the current behavior?**

**What is the new behavior (if this is a feature change)?**

**Does this PR introduce a breaking change or deprecate an API?**
- [ ] Yes
- [ ] No

**If yes, please check if the following requirements are fulfilled**

- [ ] The *Breaking Change* or *Deprecated* label has been added
- [ ] The migration steps are described in the following section

**What changes might users need to make in their application due to this PR? (migration steps)**

**Other information**:

### CODE REVIEW

**Comment by g-steiner** [comment]
I propose to add the examples that were in incubator.
I am based on implementation branch.

**Comment by annetill** [comment]
> I propose to add the examples that were in incubator. I am based on implementation branch.

You are right, my mistake... I think I also forgot the unit tests that are in the cgmes module because they need to be in the implementation module.

**Comment by g-steiner** [comment]
Actually it is in my proposed PR

File: sc-implementation/src/main/java/com/powsybl/sc/implementation/OpenShortCircuitProvider.java [hunk 1]
```diff
@@ -128,10 +128,11 @@ public void runUnbalancedAnalysis(Network network, ShortCircuitEngineParameters
 
             List<FeederResult> feederResults = new ArrayList<>();
             List<LimitViolation> limitViolations = new ArrayList<>();
-            FortescueValue current = new FortescueValue(iccMagnitude, iccAngle);
+            //FortescueValue current = new FortescueValue(iccMagnitude, iccAngle);
+            MagnitudeFaultResult magnitudeFaultResult = new MagnitudeFaultResult(fault, 0., feederResults, limitViolations, iccMagnitude, FaultResult.Status.SUCCESS);
 
-            FaultResult fr = new FortescueFaultResult(fault, 0., feederResults, limitViolations, current, FaultResult.Status.SUCCESS);
-            faultResults.add(fr);
+            //FaultResult fr = new FaultResult(fault, 0., feederResults, limitViolations, current, FaultResult.Status.SUCCESS);
```
Comment (annetill) [inline sc-implementation/src/main/java/com/powsybl/sc/implementation/OpenShortCircuitProvider.java:134]: Please clean //, I think you are right.

File: sc-implementation/src/main/java/com/powsybl/sc/implementation/OpenShortCircuitProvider.java [hunk 2]
```diff
@@ -157,8 +158,10 @@ public void runBalancedAnalysis(Network network, ShortCircuitEngineParameters sc
             List<LimitViolation> limitViolations = new ArrayList<>();
             FortescueValue current = new FortescueValue(iccMagnitude, iccAngle);
 
-            FaultResult fr = new FortescueFaultResult(fault, pcc, feederResultsProvider, limitViolations, current, FaultResult.Status.SUCCESS);
-            faultResults.add(fr);
+            MagnitudeFaultResult magnitudeFaultResult = new MagnitudeFaultResult(fault, 0., feederResultsProvider, limitViolations, iccMagnitude, FaultResult.Status.SUCCESS);
+
+            //FaultResult fr = new FaultResult(fault, pcc, feederResultsProvider, limitViolations, current, FaultResult.Status.SUCCESS);
```
Comment (annetill) [inline sc-implementation/src/main/java/com/powsybl/sc/implementation/OpenShortCircuitProvider.java:163]: Same comment.

File: sc-implementation/src/test/java/com/powsybl/sc/implementation/ShortCircuitBalancedTest.java [hunk 3]
```diff
@@ -0,0 +1,740 @@
+package com.powsybl.sc.implementation;
```
Comment (annetill) [inline sc-implementation/src/test/java/com/powsybl/sc/implementation/ShortCircuitBalancedTest.java:1]: /**
 * Copyright (c) 2022, Jean-Baptiste Heyberger & Geoffroy Jamgotchian
 * This Source Code Form is subject to the terms of the Mozilla Public
 * License, v. 2.0. If a copy of the MPL was not distributed with this
 * file, You can obtain one at http://mozilla.org/MPL/2.0/.
 */

Comment (annetill) [reply sc-implementation/src/test/java/com/powsybl/sc/implementation/ShortCircuitBalancedTest.java:1]: All new files.

**Review by annetill — APPROVED** [review]
Thanks for your contribution, we are going to merge it. @jeandemanged I imagine after that we have to delete features the incubator.

**Comment by jeandemanged** [comment]
awesome, thanks

**Pull request merged**